packages/core/src/BlockNoteEditor.ts

```diff
@@ -133,36 +121,36 @@ export class BlockNoteEditor<
     this._tiptapEditor.view.focus();
   }
 
-  constructor(options: Partial<BlockNoteEditorOptions<Block>> = {}) {
+  constructor(options: Partial<BlockNoteEditorOptions<BSchema>> = {}) {
     console.log("test");
     // apply defaults
-    options = {
+    const newOptions: Omit<typeof options, "defaultStyles" | "blockSchema"> & {
```

Comment:
It's because `options` is `Partial`, so `defaultStyles` and `blockSchema` might be undefined. Since we're applying defaults to them this changes those 2 fields to always be defined.